cmd/pulumi-resource-terraform/main.go

```diff
@@ -0,0 +1,10 @@
+package main
+
+import (
+	"github.com/pulumi/pulumi-terraform/pkg/provider"
+	"github.com/pulumi/pulumi-terraform/pkg/version"
+)
+
+func main() {
+	provider.Serve("derp", version.Version)
```

Comment:
Should this be `"terraform"`? 😉

Reply:
Yes! Somewhat surprised that still worked...